src/main/java/com/hazelcast/eurekast/one/EurekastOneDiscoveryStrategy.java

```diff
@@ -51,45 +55,94 @@
 class EurekastOneDiscoveryStrategy
```

Comment:
Yeah, i started it as a private project and eurekast was a nice mixture of eureka and hazelcast. We can go ahead and name it Hazelcast Eureka1 Discovery, just to align it with the other ones.